Ryujinx.Ava/UI/Helpers/ContentDialogHelper.cs

```diff
@@ -349,7 +349,7 @@ public static async Task<ContentDialogResult> ShowAsync(ContentDialog contentDia
 
             Window parent = GetMainWindow();
 
-            if (parent.IsActive && parent is MainWindow window && window.ViewModel.IsGameRunning)
+            if (parent is not null && parent.IsActive && parent is MainWindow window && window.ViewModel.IsGameRunning)
```

Comment:
```suggestion
            if (parent != null && parent.IsActive && parent is MainWindow window && window.ViewModel.IsGameRunning)
```
Any reason for not doing that

Reply:
not really, I'm just used to type it like that. I'll change it!

Reply:
Might also be influence from Python :D

Reply:
FYI: `is not null` generates better IL than `!=`, since the latter has to do a virtual call to `object.Equals()` while the former checks null directly

Reply:
That's good to know, thanks!